api/src/main/java/com/epam/pipeline/dao/monitoring/metricrequester/CPURequester.java

```diff
@@ -75,6 +75,7 @@ public SearchRequest buildRequest(final Collection<String> resourceIds, final Lo
     @Override
     public Map<String, Double> parseResponse(final SearchResponse response) {
         return ((Terms)response.getAggregations().get(AGGREGATION_POD_NAME)).getBuckets().stream()
+                .filter(b -> Double.isFinite(((Avg) b.getAggregations().get(AVG_AGGREGATION + USAGE_RATE)).getValue()))
```

Comment:
Probably we could remove duplicated code by using temporary `Pair` object. Something like the following. Moreover we can reuse such code in all requesters.
```java
buckets
.map(b -> Pair.of(b.getKey().toString(), ((Avg) b.getAggregations().get(AVG_AGGREGATION + NODE_UTILIZATION)).getValue()))
.filter(utilizationEntry -> Double.isFinite(utilizationEntry.getRight()))
.collect(Collectors.toMap(Pair::getLeft, Pair::getRight));
```